Sources/SwiftFusion/Core/Vector.swift

```diff
@@ -45,6 +51,9 @@ public protocol Vector: Differentiable where Self.TangentVector == Self {
   // MARK: - Methods for accessing the standard basis and for manipulating the coordinates under
   // the standard basis.
 
+  #if true // set to false to find examples to be removed.
+  // DWA TODO: Remove these requirements!
```

Comment:
Change to `TODO(marcrasi)`?

Reply:
When you have change suggestions, please use the “±” button in the web interface to propose them.  It makes them ever so much easier to accept!